README.md

```diff
@@ -765,6 +766,7 @@ JSON encoded body using the "application/json" content type.
 |bim_snippet.reference|string|Reference of a BIM-Snippet of a topic|false|
 |bim_snippet.reference_schema|string|Schema of a BIM-Snippet of a topic|false|
 |due_date|string|Until when the topics issue needs to be resolved|false|
+|guid|string|GUID of the topic. The server may ignore this and instead generate it's own guid|false|
```

Comment:
This is a duplicate of line 753. This should have been put on line 753 instead.

Reply:
Fix here: https://github.com/buildingSMART/BCF-API/pull/195. Thanks for reporting